recipe/bld.bat

```diff
@@ -0,0 +1,14 @@
+:: Generate the build files.
+cmake . -B build -G"Ninja" %CMAKE_ARGS%      ^
+    -DCMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX%  ^
+    -DCMAKE_PREFIX_PATH=%LIBRARY_PREFIX%     ^
+    -DCMAKE_BUILD_TYPE=Release
+if errorlevel 1 exit 1
+
+:: Build.
+cmake --build build -j %CPU_COUNT% --config Release
```

Comment:
```suggestion
cmake --build build -j %CPU_COUNT%
```
`--config` is for multi-configuration generators ("Visual Studio XX XXX", "Ninja Multi-Config"). See https://cmake.org/cmake/help/latest/manual/cmake.1.html#cmdoption-cmake-build-config.

Multi-configuration generators allow to generate the build system once and get multiple configurations (Release, Debug, etc) all in one go. So in a multi-config scenario, you basically do `cmake -G<generator> .` and then `cmake --build . --config Release` and you could also use `--config Debug` for example.

Reply:
I removed it also from the `install`, thank you!